server/apps/users/forms.py

```diff
@@ -0,0 +1,91 @@
+from django import forms
+
+class UserProfileForm(forms.Form):
```

Comment:
Suggested adding custom labels on the initial demographic questions to clarify what is asked compared to defaults :)

Reply:
Thanks!